Sources/WalletConnectPush/Client/Wallet/PushSubscriptionsObserver.swift

```diff
@@ -0,0 +1,22 @@
+import Combine
+import Foundation
+
+class PushSubscriptionsObserver {
```

Comment:
may better to do it inside Push SDK?

Reply:
good catch, it got to wallet files by accident 🤔

Reply:
:D it actually is in sdk already, just the wallet dir